### PR TITLE
Update swagger API documentation

### PR DESCRIPTION
**Pull request by calvinnhieu**
- serve API documentation on route `/`

### CODE REVIEW

File: config/matrix-api.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ info:
   title: HCA Matrix Service
   description: Human Cell Atlas Matrix Service API
   version: 0.0.1
-host: ${API_HOST}
+host: matrix.dev.data.humancellatlas.org
```
Comment (parthshahva) [inline config/matrix-api.yml]: why is this hardcoded?

File: chalice/Makefile [hunk 2]
```diff
@@ -30,4 +30,3 @@ deploy: clean build
 	@echo -e "#########################################################\n"
 	./build_deploy_config.sh $(DEPLOYMENT_STAGE)
 	chalice deploy --no-autogen-policy --stage $(DEPLOYMENT_STAGE) --api-gateway-stage $(DEPLOYMENT_STAGE)
-	rm -rf chalicelib
```
Comment (mckinsel) [inline chalice/Makefile]: What's this bit about?

File: chalice/app.py [hunk 3]
```diff
@@ -49,14 +49,14 @@ def dispatch(*args, **kwargs):
...
-    # with open(os.path.join(pkg_root, "index.html")) as fh:
-    #     swagger_ui_html = fh.read()
-    #
-    # @app.route("/")
-    # def serve_swagger_ui():
-    #     return chalice.Response(status_code=200,
-    #                             headers={"Content-Type": "text/html"},
-    #                             body=swagger_ui_html)
+    with open(os.path.join(pkg_root, "index.html")) as fh:
+        swagger_ui_html = fh.read()
```
Comment (mckinsel) [inline chalice/app.py]: Does chalice make the "index.html"?

File: chalice/Makefile (hunk 2 above)
Comment (calvinnhieu) [reply chalice/Makefile]: I added this awhile ago to keep the working dir clean, but removing it now to preserve `chalicelib/index.html` (same pattern as other repos)

File: chalice/app.py (hunk 3 above)
Comment (calvinnhieu) [reply chalice/app.py]: No, I looked at Upload's `index.html` and it's a template with a bunch of CSS that populates the content by reading in a Swagger yaml, so I copied it over.

Comment (calvinnhieu) [reply chalice/app.py]: I would think that Swagger has a tool to generate the `index.html`, but I don't know where/what that is

**Pull request merged**